src/secure_random.cr

```diff
@@ -1,20 +1,60 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom module is an interface for creating secure random values in different formats. 
+# It uses the RNG (random number generator) of LibCrypto (OpenSSL).
+#
+# For example:
+# ```crystal
+# SecureRandom.base64 #=> "LIa9s/zWzJx49m/9zDX+VQ=="
+# SecureRandom.hex    #=> "c8353864ff9764a39ef74983ec0d4a38"
+# SecureRandom.uuid   #=> "c7ee4add-207f-411a-97b7-0d22788566d6"
+# ```
 module SecureRandom
-  def self.base64(n = 16 : Int)
+  # Generates an ASCII string encoded with Base64.
```

Comment:
It generates random bytes that are encoded into Base64. The result of encoding something into Base64 (and most of the time the entire point of doing so) is an ASCII string.